data/json/recipes/armor/torso.json

```
@@ -2631,5 +2631,27 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "bag_garbage", 1 ] ] ],
     "flags": [ "BLIND_HARD" ]
+  },
+  {
+    "type": "recipe",
+    "result": "suspenders_cotton",
+    "skill_used": "tailor",
+    "difficulty": 1,
+    "time": "1 h",
+    "autolearn": true,
+    "reversible": true,
+    "using": [ [ "rag", 5 ], [ "clasps", 3 ] ],
```

```suggestion
    "using": [ [ "tailoring_cotton", 1 ], [ "clasps", 3 ], [ "fabric_standard_nostretch", 5 ] ],
```

Not super sure about this one, but "rag" is not a valid requirement. It also needs some kind of tool requirement ("tailoring_cotton").